setup.py

```diff
@@ -23,22 +23,23 @@ def parse_version(module_file):
     return match[0]
 
 f = open("README.txt")
-ga4gh_readme = f.read()
+ga4ghReadme = f.read()
 f.close()
-ga4gh_version = parse_version("ga4gh/__init__.py")
+ga4ghVersion = parseVersion("ga4gh/__init__.py")
 # Flask must come after all other requirements that have "flask" as a prefix
 # due to a setuptools bug.
-requirements = ["avro", "Flask-API", "flask-cors", "flask", "pysam", "requests", "wormtable"]
+requirements = ["avro", "Flask-API", "flask-cors", "flask", "pysam",
+                "requests", "wormtable"]
 v = sys.version_info[:2]
```

Comment:
Maybe we should delete these 3 lines also? I doubt we're every going to support Python 2.6 or 3.1 (TravisCI have stopped supporting 3.1) and we definitely don't need to worry about supporting 3.0. Sorry, I know this was just a quick pep8 update - feel free to ignore this if you don't want to deal with this now!